File: src/Salon.Domen/Entiteti/Usluga.cs
namespace Salon.Domen.Entiteti;

public class Usluga
{
    public int Id { get; set; }

    public string Naziv { get; set; } = string.Empty;

    public string Opis { get; set; } = string.Empty;

    public int TrajanjeUMinutima { get; set; }

    public int MaksimalanBrojKlijenataPoTerminu { get; set; }

    public TimeSpan VremePocetkaPrvogTermina { get; set; }

    public TimeSpan VremeZavrsetkaPoslednjegTermina { get; set; }

    public decimal Cena { get; set; }

    public int KategorijaUslugeId { get; set; }

    public KategorijaUsluge KategorijaUsluge { get; set; } = null!;
}

File: src/salon-web/src/models/Usluga.ts
export interface Usluga {
  id: number
  naziv: string
  opis: string
  trajanjeUMinutima: number
  maksimalanBrojKlijenataPoTerminu: number
  vremePocetkaPrvogTermina: string
  vremeZavrsetkaPoslednjegTermina: string
  cena: number
  kategorijaUslugeId: number
  kategorijaNaziv?: string | null
}